graphql/schema.graphql

```diff
@@ -487,6 +486,7 @@ type Patch {
   status: String!
   variants: [String!]!
   tasks: [String!]!
+  childPatches: [ChildPatches!]!
```

Comment:
I did this based on Arjun's suggestion. @SupaJoon, thoughts?

Reply:
Maybe he meant `[ChildPatch!]` ? That implies that it will always have at least 1 ChildPatch or it i will return null.

`[ChildPatch]` implies something like this is possible `[null]` 
`[ChildPatch!]!` implies that `[{some:thing}] | []`
`[ChildPatch!]` implies that `[{some:thing}] | null`

Reply:
[ChildPatch!] is helpful if we need distinguish between null or [] in the UI. Otherwise i think it's easier to resemble "emptiness" with an empty array so we have to deal with less types in the UI. I also suggested this change bc I saw that the resolver will not return null for this value unless an error occurred.

Reply:
On the other hand our API will read more expressive if we indicate emptiness by returning null and omitting `!` in the schema